src/app/projects/projects.service.ts

```diff
@@ -35,7 +35,7 @@ export class ProjectsService {
     try {
       return {
         uuid: obj.uuid.uuid,
-        dcpUrl: obj.dcp_url,
+        dcpUrl: obj.wranglingState === 'Published in DCP' && `https://data.humancellatlas.org/explore/projects/${obj.uuid.uuid}`,
```

Comment:
How does this work? does this return the link if the first condition is true?

Reply:
Yes and null if not

Reply:
cool! I didn't know that!